src/operators.jl

```diff
@@ -41,6 +43,11 @@ ops_1_to_1_s = (
     # other
     :inv,
 )
+for op in ops_1_to_1_s
+    T = typeof(eval(op))
+    SparseConnectivityTracer.is_firstder_zero_global(::T) = false
```

Comment:
why prefix with the package name every time?

Reply:
To define methods in the scope of the module. Is there another way? Removing the module name, I get the error 
```
ERROR: LoadError: UndefVarError: `T` not defined
Stacktrace:
 [1] top-level scope
   @ ~/Developer/SparseConnectivityTracer.jl/src/operators.jl:46
```

Reply:
I spontaneously would have tried something like
```julia
@eval is_firstder_yero_global(::$(typeof(op)))
```
and manipulated the actual operations instead of symbols. But that's very minor and can easily be fixed so don't let it stop you